collector/exporter/otelarrowexporter/README.md

```diff
@@ -171,14 +171,57 @@ of data being returned to the exporter will be instrumented:
 - `exporter_recv`: uncompressed bytes received, prior to compression
 - `exporter_recv_wire`: compressed bytes received, on the wire.
 
-### Experimental Configuration
+### Compression Configuration
+
+The exporter supports configuring Zstd compression at both the gRPC
+and the Arrow level.  The exporter metrics described above will be
+correct in either case.  The default settings are subject to change as
+we gain experience.
+
+The gRPC-level Zstd compression can be configured, however there is an
+important caveat.  The gRPC-Go library requires that compressor
+implementations be registered statically.  These libraries use
+compressors named `zstdarrow1`, `zstdarrow2`, ..., `zstdarrow10`,
+supporting 10 configurable compression levels.  Note, however that
+these configurations are static and only one unique configuration is
+possible per level.  It is possible to configure multiple OTel-Arrow
+exporters with different Zstd configuration simply by using distinct
+levels.
+
+Under `arrow`, the `zstd` sub-configuration has the following fields:
+
+- `level`: in the range 1-10 determines a number of defaults (default 5)
+- `window_size_mib`: size of the Zstd window in MiB, 0 indicates to determine based on level (default 0)
+- `concurrency`: controls background CPU used for compression, 0 indicates to let `zstd` library decide (default 1)
 
 The exporter supports configuring compression at the [Arrow
 columnar-protocol
 level](https://arrow.apache.org/docs/format/Columnar.html#format-ipc).
 
 - `payload_compression`: compression applied at the Arrow IPC level, "none" by default, "zstd" supported.
```

Comment:
Hum I thought that we used zstd compression at the arrow level by default.

Reply:
I have evidence that by configuring the Zstd level and window size in the encoder, the payload compression is not actually helping very much. All of our earlier experiments w/ gRPC-level Zstd compression had inadvertently used a relatively small window size, which defeated the gRPC-level compression and made Arrow-level payload compression relatively important.

Coincidentally, I audited the Arrow code that sets up Zstd and recognize that it is using single-record encode and decode operations, there are no Zstd streams being built up and there is also no configurability. I wonder whether the Arrow architects have considered using Zstd streams within the Arrow IPC streams, but it seems to not be that way presently.

Reply:
Since https://github.com/open-telemetry/otel-arrow/pull/82 merged I will repeat my experiment that led to this conclusion.

Reply:
Sounds good.

Reply:
Confirming that for the 30GB uncompressed data sample you and I have been using, @lquerel, for a single stream lasting the entire input (with #82), single-zstd at the gRPC level outperforms double-zstd at various levels.

For the uncompressed size 2.9875869937e+10, the results:

single-zstd, level 10: 2.540397625e+09 i.e., 0.085
double-zstd, level 10: 2.663619163e+09 i.e., 0.089
single-zstd, level 8: 2.627058159e+09 i.e., 0.088
single-zstd, level 5: 2.701691178e+09 i.e., 0.090
double-zstd, level 5: 2.870220537e+09, i.e., 0.096